open_lm/params.py

```diff
@@ -742,6 +742,11 @@ def parse_args(args):
         action="store_true",
         help="If set, allow model to do multiple data passes over our dataset, in order to reach the desired number of tokens.",
     )
+    parser.add_argument(
+        "--mask-across-documents",
```

Comment:
i think this should be an `int` not a `bool` so that a user can specify their EOT token

Reply:
Makes sense - will update the parameter.